lib/snprintf.c

```diff
@@ -1097,6 +1097,9 @@ fmtflt(char *str, size_t *len, size_t size, LDOUBLE fvalue, int width,
 		infnan = (flags & PRINT_F_UP) ? "INF" : "inf";
 
 	if (infnan != NULL) {
+        int i = MAX_CONVERT_LENGTH - 1;
+        while (i >= 0)
+            iconvert[i--] = '\0';
```

Comment:
Why not simply memset()?

Reply:
Oh, I now see your response in an email from GitHub. I'd change to using memset(), then I'll merge in these changes.